internal/install/_static/Dockerfile.terraform_deployer

```diff
@@ -1,17 +1,14 @@
 FROM hashicorp/terraform:light as terraform
```

Comment:
Now that we have apt maybe we can also install terraform using it :) https://www.terraform.io/downloads

And we could also pin the version.

This is unrelated to this PR in any case.

Reply:
Actually it may help if this is a PATH issue or library issue :)

Reply:
Added installation through deb thanks for the suggestion!